tests/system/action/committee/test_json_upload.py

```diff
@@ -0,0 +1,298 @@
+            "messages": [],
+            "data": {"name": "test", "id": 7},
+        }
+
+    def test_json_upload_date(self) -> None:
+        response = self.request(
+            "committee.json_upload",
+            {
+                "data": [
+                    {
+                        "name": "test",
+                        "meeting_name": "test meeting",
+                        "start_date": "2023-08-09",
+                        "end_date": "2023-08-10",
+                    }
+                ]
+            },
+        )
+        self.assert_status_code(response, 200)
+        assert response.json["results"][0][0]["rows"][0] == {
+            "state": ImportState.NEW,
+            "messages": [],
+            "data": {
+                "name": "test",
+                "meeting_name": "test meeting",
+                "start_date": 1691539200,
+                "end_date": 1691625600,
+            },
+        }
+
+    def test_json_upload_wrong_date(self) -> None:
+        response = self.request(
+            "committee.json_upload",
+            {
+                "data": [
+                    {
+                        "name": "test",
+                        "meeting_name": "test meeting",
+                        "start_date": "2023-08-09",
+                        "end_date": "12XX-broken",
+                    }
+                ]
+            },
+        )
+        self.assert_status_code(response, 400)
+        assert "Could not parse 12XX-broken except date" in response.json["message"]
+
+    def test_json_upload_meeting_field_but_no_meeting_name(self) -> None:
+        response = self.request(
+            "committee.json_upload",
+            {
+                "data": [
+                    {
+                        "name": "test",
+                        "meeting_template": "testtemplate",
+                    }
+                ]
+            },
+        )
+        self.assert_status_code(response, 200)
+        assert response.json["results"][0][0]["rows"][0] == {
+            "state": ImportState.ERROR,
+            "messages": ["Meeting field given, but no meeting_name"],
+            "data": {
+                "name": "test",
+                "meeting_template": {
+                    "value": "testtemplate",
+                    "info": ImportState.WARNING,
+                },
+            },
+        }
+
+    def test_json_upload_meeting_template_not_found(self) -> None:
+        response = self.request(
+            "committee.json_upload",
+            {
+                "data": [
+                    {
+                        "name": "test",
+                        "meeting_name": "test meeting",
+                        "meeting_template": "test",
+                    }
+                ]
+            },
+        )
+        self.assert_status_code(response, 200)
+        assert response.json["results"][0][0]["rows"][0] == {
+            "state": ImportState.NEW,
+            "messages": [],
```

Comment:
Missing message, that the meeting_template wasn't found and on import a meeting will be created via meeting.create action.

Reply:
Add the missing meeting_template message.